examples/generative/corrdiff/train.py

```diff
@@ -207,19 +212,34 @@ def main(cfg: DictConfig) -> None:
     ## Resume training from previous checkpoints if exists
     if dist.world_size > 1:
         torch.distributed.barrier()
-    cur_nimg = load_checkpoint(
-        path="checkpoints", models=model, optimizer=optimizer, device=dist.device
-    )
+    try:
+        cur_nimg = load_checkpoint(
+            path=f"checkpoints_{cfg.model.name}",
+            models=model,
+            optimizer=optimizer,
+            device=dist.device,
+        )
+    except:
+        cur_nimg = 0
+
+    def is_time_for_periodic_task(freq, rank_0_only=False):
```

Comment:
Maybe consider moving this to `training_helpers.py`

Reply:
Done in https://github.com/NVIDIA/modulus/pull/636/commits/3df9d26f94ec5d13075e1bd6b1f919a9b07b114c.